### ISSUE TITLE
The hamburger menu 'collapse' button in Bootstrap needs an aria label

### ISSUE THREAD

**Issue by ronm123** [thexerteproject/xerteonlinetoolkits#935]
This shows up as an accessibility issue because the collapse button doesn't have an aria label. Can be temporary fixed within an LO using something like `$(".btn.btn-navbar").attr("aria-label", "collapse");` but ideally we need to add/insert an aria label by default.
